erpnext/controllers/selling_controller.py

```diff
@@ -40,6 +40,8 @@ def set_missing_values(self, for_validate=False):
 		super(SellingController, self).set_missing_values(for_validate)
 
 		# set contact and address details for customer, if they are not mentioned
+		# set default taxes
+		self.set_other_charges()
```

Comment:
@rohitwaghchaure Why so? This code applies to all sales transactions which is not intended.